crates/autopilot/src/database.rs

```diff
@@ -92,7 +104,7 @@ pub async fn database_metrics(db: Postgres) -> ! {
         if let Err(err) = db.update_database_metrics().await {
             tracing::error!(?err, "failed to update table rows metric");
         }
-        tokio::time::sleep(Duration::from_secs(60)).await;
+        tokio::time::sleep(Duration::from_secs(60 * 5)).await;
```

Comment:
One more proposal is to make updates less regular.

Reply:
Since the estimation query is in the order of 200ms now I think having more recent metrics is more important than saving a few DB resources.

Reply:
Counting rows in other tables takes a couple of seconds(2-7).